Source/Classes/PINRemoteImageManager.m

```diff
@@ -1356,7 +1419,15 @@ - (BOOL)materializeAndCacheObject:(id)object
     }
     
     if (diskData) {
-        [self.cache setObjectOnDisk:diskData forKey:key];
+        // maxAge of 0 is not stored at all
```

Comment:
Could you clarify this comment? Does this mean "maxAge of 0 means that images should not be stored at all" ?

Is there a value for maxAge that means infinite?

Reply:
Done (yes that's what it meant).
There is no header that can be sent to indicate "infinite", which is why you will see so many HTTP responses with values like "one year in the future". However not setting a value at all, which in our case is represented by `maxAge == nil`, does effectively mean that. (In our case it will use the cache-global maxAge)